addons/web/static/src/views/helpers/sample_server.js

```diff
@@ -118,8 +118,8 @@ export class SampleServer {
         // global (with key 'method')
         const method = params.method || params.route;
         const mockFunction =
-            SampleServer.mockRegistry.get(`${params.model}/${method}`) ||
-            SampleServer.mockRegistry.get(method);
+            registry.category("sample_server").get(`${params.model}/${method}`, null) ||
```

Comment:
// mockRegistry allows to register mock version of methods or routes,
// for all models:
//   SampleServer.mockRegistry.add('some_route', () => "abcd");
// for a specific model (e.g. 'res.partner'):
//   SampleServer.mockRegistry.add('res.partner/some_method', () => 23);